### PR TITLE
fix first page button for uncles and reorgs

### PR DESCRIPTION
**Pull request by ayrat555**
fixes https://github.com/poanetwork/blockscout/issues/2556

## Changelog

- fix the first-page button for uncles and reorgs

### CODE REVIEW

File: apps/block_scout_web/assets/js/lib/async_listing_load.js [hunk 1]
```diff
@@ -205,7 +205,15 @@ export const elements = {
       }
       $el.show()
       $el.attr('disabled', false)
-      $el.attr('href', window.location.href.split('?')[0])
+
+      const urlParams = new URLSearchParams(window.location.search)
+      const blockParam = urlParams.get('block_type')
+
+      if (blockParam !== null) {
+        $el.attr('href', window.location.href.split('?')[0] + '?block_type=' + blockParam)
```
Comment (vbaranov) [inline apps/block_scout_web/assets/js/lib/async_listing_load.js]: @ayrat555 let's define a const for `window.location.href.split('?')[0]`

Comment (ayrat555) [reply apps/block_scout_web/assets/js/lib/async_listing_load.js]: @vbaranov done

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 ### Features
 
 ### Fixes
+- [#2556](https://github.com/poanetwork/blockscout/issues/2556) - fix first page button for uncles and reorgs
```
Comment (pasqu4le) [inline CHANGELOG.md]: PR number is wrong, should be 2564 instead

Comment (saneery) [reply CHANGELOG.md]: And the link should be to the PR not to an issue

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 ### Features
 
 ### Fixes
+- [#2564](https://github.com/poanetwork/blockscout/issues/2564) - fix first page button for uncles and reorgs
```
Comment (saneery) [inline CHANGELOG.md]: I think the link should be to the PR not to the issue

**Pull request merged**